src/layers/core/screen-grid-layer/screen-grid-layer-vertex.glsl

```diff
@@ -27,20 +27,24 @@
 
 attribute vec3 vertices;
 attribute vec3 instancePositions;
-attribute vec4 instanceColors;
+attribute float instanceCount;
 attribute vec3 instancePickingColors;
 
 uniform float maxCount;
 uniform float opacity;
+uniform vec4 minColor;
+uniform vec4 maxColor;
 uniform float renderPickingBuffer;
 uniform vec3 cellScale;
 uniform vec3 selectedPickingColor;
 
 varying vec4 vColor;
 
 void main(void) {
+  vec4 instanceColor = mix(minColor, maxColor, instanceCount / maxCount);
```

Comment:
This temporary variable`instanceColor`, made me think it was an attribute, I had to read the code two times. Maybe just `color` would be simple?